src/V3Options.cpp

```diff
@@ -853,19 +853,18 @@ void V3Options::notify() VL_MT_DISABLED {
                 + ". Suggest see manual");
     }
 
-    if (m_exe && !v3Global.opt.libCreate().empty()) {
-        cmdfl->v3error("--exe cannot be used together with --lib-create. Suggest see manual");
-    }
-
```

Comment:
I don't see how this patch makes that work?  If if it does can that be a separate pre-pull?

Reply:
This change will be useful in more complex hierarchical setup, as for now not used thus I removed it.